src/input-helper.ts

```diff
@@ -22,9 +22,16 @@ export function getInputs(): UploadInputs {
     )
   }
 
-  return {
+  const inputs = {
     artifactName: name,
     searchPath: path,
     ifNoFilesFound: noFileBehavior
+  } as UploadInputs
+
+  const retentionDaysStr = core.getInput(Inputs.RetentionDays)
+  if (retentionDaysStr) {
+    inputs.retentionDays = parseInt(retentionDaysStr)
```

Comment:
I think some extra validation is needed. `parseInt` can return `NaN` if the input is not a number.

Reply:
Added the check for `isNaN`.  Everything else should be checked by the toolkit itself.